mappings/net/minecraft/world/gen/stateprovider/NoiseCutoffBlockStateProvider.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_6581 net/minecraft/world/gen/stateprovider/NoiseCutoffBlockStateProvider
```

Comment:
Its registry name is `noise_threshold_provider`, so I think maybe using `Threshold` than `Cutoff` in name may be better.

Reply:
It was `cutoff` in 21w37a, I didn't see the rename.

Reply:
Yeah, things look good beyond that.